Client/FrameBuffer.cpp

```diff
@@ -0,0 +1,124 @@
+﻿#include "FrameBuffer.h"
+#include <glad/glad.h>
+#include <glm/glm.hpp>
+#include <memory>
+#include "Shader.hpp"
+
+static const GLfloat g_quad_vertex_buffer_data[] = {
+  -1.0f, -1.0f, 0.0f,
+  1.0f, -1.0f, 0.0f,
+  -1.0f, 1.0f, 0.0f,
+  -1.0f, 1.0f, 0.0f,
+  1.0f, -1.0f, 0.0f,
+  1.0f, 1.0f, 0.0f,
+};
+
+FrameBuffer::FrameBuffer() : FrameBuffer(1024, 1024) {
+
+}
+
+FrameBuffer::FrameBuffer(int width, int height) {
```

Comment:
I'm thinking it might be good to break the code in the constructor up into helper functions just to make it a bit more readable and easily modifiable. I don't think it should be a huge deal to keep it like this, though, so it's up to you.

Reply:
I think it is fine this way for now, we probably won't have a lot to change in this part of code.

Reply:
I didn't use helper functions, because most likely nowhere else will reuse this code. If you think it's needed, I can break it down.

Reply:
Sounds good, I think we're fine keeping it the way it is.